src/shared/components/dateRangePicker/DatePicker.tsx

```diff
@@ -38,25 +39,25 @@ interface State {
 
 const isValidRTC3339 = (d: string): boolean => {
   return (
-    moment(d, 'YYYY-MM-DD HH:mm', true).isValid() ||
-    moment(d, 'YYYY-MM-DD HH:mm:ss', true).isValid() ||
-    moment(d, 'YYYY-MM-DD HH:mm:ss.SSS', true).isValid() ||
-    moment(d, 'YYYY-MM-DD', true).isValid() ||
-    moment(d).toISOString() === d
+    isValid(d, 'YYYY-MM-DD HH:mm') ||
+    isValid(d, 'YYYY-MM-DD HH:mm:ss') ||
+    isValid(d, 'YYYY-MM-DD HH:mm:ss.SSS') ||
+    isValid(d, 'YYYY-MM-DD') ||
+    new Date(d).toISOString() === d
```

Comment:
Technically wasn't an `isValid` function change, but slipped this in.

Reply:
Had to put it back, there is some stuff with datepicker that would need its own issue and PR.